### PR TITLE
Implement expand/collaspe all button for publication page.

### PR DESCRIPTION
**Pull request by PorametPat**
Add a simple button to toggle between expand and collapse all contents within publication page.

### CODE REVIEW

File: src/app/publications/page.tsx [hunk 1]
```diff
@@ -135,6 +151,7 @@ export default function Page() {
             </AccordionItem>
           ))}
         </Accordion>
+        B
```
Comment (zigen) [inline src/app/publications/page.tsx:154]: typo or something? could you remove it?

**Review by zigen — CHANGES_REQUESTED** [review]

**Comment by zigen** [comment]
@PorametPat ping

File: src/app/publications/page.tsx (hunk 1 above)
Comment (PorametPat) [reply src/app/publications/page.tsx:154]: I think it is typo.

**Comment by PorametPat** [comment]
@zigen Ping

**Review by zigen — APPROVED** [review]

**Pull request merged**